addons/html_editor/static/src/core/zws_plugin.js

```diff
@@ -31,6 +31,13 @@ export class ZwsPlugin extends Plugin {
                     break;
             }
         });
+        this.addDomListener(this.editable, "input", (ev) => {
```

Comment:
Be sure to check (and unit test) if this change won't be added in a separate history step.
 I personally would prefer such cleaning to happen on NORMALIZE.